.travis.yml

```diff
@@ -38,6 +34,27 @@ matrix:
       # No version of Python is available via virtualenv on OS X workers, see https://github.com/travis-ci/travis-ci/issues/2312
       language: generic
       env: TOX_ENV=py27-first_startup
+    - env: js
+      language: node_js
+      node_js:
+          - 10
+      before_install:
+        - cd client
+      install:
+        - yarn
+        - yarn run build
+      script:
+        - yarn run test
+    - env: js
```

Comment:
Maybe this job can be merged with the previous one?

Reply:
We *could*, but I think I'd prefer to keep it separate, like the python jobs.  I'm going to update the name though to indicate lint vs test.

Reply:
Fair enough, thanks!